backend/program/libaries/symlink.py

```diff
@@ -19,7 +19,21 @@ def __init__(self):
         self.last_fetch_times = {}
         self.settings = settings_manager.settings.symlink
         self.initialized = True
-
+        self.initialized = self.validate()
+        if not self.initialized:
+            logger.error("SymlinkLibrary initialization failed due to invalid configuration.")
+            return
+    
+    def validate(self) -> bool:
+        status = all(
+            os.path.exists(self.settings.library_path / d) 
+            for d in ( "shows", "movies", "anime_shows", "anime_movies")
```

Comment:
You probably noticed by now, but we dont technically use the anime dir's just yet. I added a `is_anime` attr that we can use to populate those 2 dirs though.

Just gotta add sorting to this module to handle

Reply:
We don't populate the library from them either at the moment

Reply:
we can revisit it later when we're more stable.. but would like to sort anime out of the main 2 libraries.